audit/projects/k8s-infra-e2e-boskos-001/services/logging/logs.json

```diff
@@ -1,132 +0,0 @@
-[
-  "projects/k8s-infra-e2e-boskos-001/logs/GCEGuestAgent",
-  "projects/k8s-infra-e2e-boskos-001/logs/OSConfigAgent",
-  "projects/k8s-infra-e2e-boskos-001/logs/affinity-clusterip",
-  "projects/k8s-infra-e2e-boskos-001/logs/affinity-clusterip-timeout",
```

Comment:
Expected, this is a "filter out log churn noise from k8s-infra-e2e-*" fix from https://github.com/kubernetes/k8s.io/pull/2133